k8s/pkg/apis/longhorn/v1beta2/orphan.go

```diff
@@ -5,12 +5,15 @@ import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 type OrphanType string
 
 const (
-	OrphanTypeReplica = OrphanType("replica")
+	OrphanTypeReplicaDataStore = OrphanType("replica")
+	OrphanTypeEngineInstance   = OrphanType("engine-instance")
+	OrphanTypeReplicaInstance  = OrphanType("replica-instance")
 )
 
 const (
-	OrphanConditionTypeError         = "Error"
-	OrphanConditionTypeDataCleanable = "DataCleanable"
+	OrphanConditionTypeError                = "Error"
+	OrphanConditionTypeDataCleanable        = "DataCleanable"
+	OrphanConditionTypeRuntimeInstanceState = "InstanceState"
```

Comment:
I think we can remove `Runtime`. `Instance` should be enough and is already used in longhorn-manager and other componenets.

```suggestion
	OrphanConditionTypeInstanceState = "InstanceState"
```